typescript/README.md

```diff
@@ -0,0 +1,14 @@
+## Substreams Sink Examples - JS
+
+The [Substreams JS library](https://github.com/substreams-js/substreams-js) allows you to consume Substreams packages using the Typescript programming language. The library works on both NodeJS and the Browser, but with some differences.
+
+### Persisting the Cursor
+
+Consuming a Substreams package involves opening a stream long-live gRPC connection. Disconnections will happen and it is necessary to create a reconnection mechanism that starts reading exactly where the stream was interrupted. The **cursor** provided by Substream must be persisted and, in the case of a disconnection, the latest committed cursor must be used.
+
+You can read more about this topic in the [Substreams docs](https://substreams.streamingfast.io/documentation/consume/reliability-guarantees).
+
+### NodeJS vs Browser
+
+- **Libraries:** running Substreams JS on NodeJS requires using the `@connectrpc/connect-node` library, while consuming packages on the browser requires using `@bufbuild/connect-web`.
```

Comment:
The up to date dependency for web should be something like `@connectrpc/connect-web`.

`@bufbuild` should only be used for "pure" Protobuf things., anything gRPC related should be coming from one of the `@connectrpc` package

Reply:
`@connectrpc/connect-web` as found in the `package.json` definition